hub/src/server/authentication.js

```diff
@@ -359,16 +359,31 @@ export function parseAuthHeader(authHeader: string) {
   }
 }
 
+export function isBlackListedAuthorization(authHeader: string, blacklist: ?Map<string, boolean>) {
+  if (!blacklist)
+    return
+  if (!authHeader || !authHeader.toLowerCase().startsWith('bearer')) {
+    throw new ValidationError('Failed to parse authentication header.')
+  }
+  const authPart = authHeader.slice('bearer '.length)
+  if (blacklist.has(authPart)) {
```

Comment:
Because we're strictly making the `blacklist` Map from an array, they'll always have a value of `true`. But `Map.has(key)` will return true, even if the blacklist has the value `key: false`. It's a little confusing here.

Maybe use a `Set`?

~~~javascript
blacklist = new Set()
blacklist.add('key')
blacklist.has('key)
~~~

Reply:
Ah! I didn't know about `Set`s, that's exactly what I wanted.